unit/test/Get-VSTeamStaleBranch.Tests.ps1

```diff
@@ -0,0 +1,168 @@
+Set-StrictMode -Version Latest
+
+InModuleScope VSTeam {
+
+   # Set the account to use for testing. A normal user would do this
+   # using the Set-VSTeamAccount function.
+   [VSTeamVersions]::Account = 'https://dev.azure.com/test'
+
+   $results = [PSCustomObject]@{
+      Branch = [PSCustomObject]@{
+         objectId = '6f365a7143e492e911c341451a734401bcacadfd'
+         name     = 'refs/heads/master'
+         creator  = [PSCustomObject]@{
+            displayName = 'Microsoft.VisualStudio.Services.TFS'
+            id          = '1'
+            uniqueName  = 'some@email.com'
+         }
+      }
+      Creator = [PSCustomObject]@{
+         displayName = 'Microsoft.VisualStudio.Services.TFS'
+         id          = '1'
+         uniqueName  = 'some@email.com'
+      }
+      CreationDate = "1010101"
+      ProjectName = "Test"
+      Repository = "repo"
+      LastCommit = "Block-SmbShareAccess"
+      LastCommitter = [PSCustomObject]@{
+         displayName = 'Microsoft.VisualStudio.Services.TFS'
+         id          = '1'
+         uniqueName  = 'some@email.com'
+      }
+   }
+
+   $singleRepository = [VSTeamGitRepository]::new([PSCustomObject]@{
```

Comment:
When you have complex objects for tests. You might want to save an example JSON in the subfolder (examplefiles](https://github.com/DarqueWarrior/vsteam/tree/master/unit/test/sampleFiles). This makes the unit test simpler to understand and also reusable if other unit tests need to use it.

Reply:
True, will see that I can do that in my next revision